python/lsst/ts/mtaos/config_schema.py

```diff
@@ -383,30 +382,43 @@
         initialCutoutPadding:
           type: integer
           default: 40
+calcZernikesTask:
+  type: object
+  additionalProperties: false
+  properties:
+    class:
+      type: string
+      default: lsst.ts.wep.task.calcZernikesTask.CalcZernikesTask
 """
 )
 
 CWFS_PIPELINE_CONFIG = yaml.safe_load(
-    """estimateZernikesCWFSTask:
-    type: object
-    additionalProperties: false
-    properties:
-      class:
-        type: string
-        default: >-
-          lsst.ts.wep.task.EstimateZernikesCwfsTask.EstimateZernikesCwfsTask
-      config:
-        type: object
-        additionalProperties: false
-        properties:
-          donutTemplateSize:
-            type: integer
-            default: 160
-          donutStampSize:
-            type: integer
-            default: 160
-          initialCutoutPadding:
-            type: integer
-            default: 40
+    """cutOutDonutsCwfsTask:
+  type: object
+  additionalProperties: false
+  properties:
+    class:
+      type: string
+      default: lsst.ts.wep.task.cutOutDonutsCwfsTask.CutOutDonutsCwfsTask
+    config:
+      type: object
+      additionalProperties: false
+      properties:
+        donutTemplateSize:
+          type: integer
+          default: 160
```

Comment:
I thought we stopped providing defaults in our config schemas?

Reply:
Good question! You are correct that for CSC configuration we should not provide defaults in the schema, but in the `_init.yaml` file. However, these schemas here are not "CSC configuration"... They are schemas for the wavefront estimation pipeline.